### DISCUSSION TITLE
How do I install starship on MacOS?

### DISCUSSION THREAD

**Discussion by kaihendry** [starship/starship#5579] [Q&A] [answered]
On a vanilla MacOS machine, I `brew install starship` and configured my bash shell with eval "$(starship init bash)". It runs, but it's missing nerd fonts. 

I tried:

```
brew tap homebrew/cask-fonts
brew install font-fira-code-nerd-font
starship preset nerd-font-symbols > ~/.config/starship.toml
```

But that didn't work and worse is that my starship.toml was filled with junk. Is there a saner quickstart please?

**Comment by davidkna** [comment]
Have you changed your terminal font to Fira Code NF yet?

**Reply by kaihendry** [reply]
Not sure how you do that on VScode terminal

**Reply by davidkna** [reply, marked as answer]
It's the terminal.integrated.fontFamily setting.

**Reply by kaihendry** [reply]
`brew install font-meslo-lg-nerd-font` and configurating "MesloLGM Nerd Font" WFM
